migrator/migrations/m_214_to_m_215_populate_alert_deployment_containers_imageidv2/migration_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build sql_integration
+
+package m214tom215
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations/m_214_to_m_215_populate_alert_deployment_containers_imageidv2/schema"
+	"github.com/stackrox/rox/migrator/migrations/m_214_to_m_215_populate_alert_deployment_containers_imageidv2/store"
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/fixtures/fixtureconsts"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/protoassert"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stretchr/testify/suite"
+)
+
+type migrationTestSuite struct {
+	suite.Suite
+
+	store store.Store
+	db    *pghelper.TestPostgres
+	ctx   context.Context
+}
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(migrationTestSuite))
+}
+
+func (s *migrationTestSuite) SetupSuite() {
+	s.ctx = sac.WithAllAccess(context.Background())
+	s.db = pghelper.ForT(s.T(), false)
+	pgutils.CreateTableFromModel(s.ctx, s.db.GetGormDB(), schema.CreateTableAlertsStmt)
```

Comment:
You can add an option to scale test using an existingDB similar to the process indicator migration here:
https://github.com/stackrox/stackrox/blob/master/migrator/migrations/m_212_to_m_213_add_container_start_column_to_indicators/migration_test.go#L37-L40